Lesson1/Task3/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter first row: ");
+            string row1 = Console.ReadLine();

```

Comment:
@okuzmina Just for you information: you could make code more compact, using sequential function call, for example for the first row it could be:
 string row1Upper = Console.ReadLine().ToUpper();